src/main/java/se/sundsvall/supportmanagement/service/config/EmailIntegrationConfigService.java

```diff
@@ -15,14 +17,19 @@ public class EmailIntegrationConfigService {
 
 	private EmailWorkerConfigRepository configRepository;
 	private EmailIntegrationMapper mapper;
+	private NamespaceConfigRepository namespaceConfigRepository;
 
 
-	public EmailIntegrationConfigService(EmailWorkerConfigRepository configRepository, EmailIntegrationMapper mapper) {
+	public EmailIntegrationConfigService(EmailWorkerConfigRepository configRepository, EmailIntegrationMapper mapper, NamespaceConfigRepository namespaceConfigRepository) {
 		this.configRepository = configRepository;
 		this.mapper = mapper;
+		this.namespaceConfigRepository = namespaceConfigRepository;
 	}
 
 	public void create(EmailIntegration request, String namespace, String municipalityId) {
+		if(namespaceConfigRepository.getByNamespaceAndMunicipalityId(namespace, municipalityId).isEmpty()) {
+			throw Problem.valueOf(INTERNAL_SERVER_ERROR, "Namespace config must be created before enabling email integration. Add via /namespaceConfig resource");
+		}
```

Comment:
Not sure if it really matters but using a existsByNamespaceAndMunicipalityId eliminates unnecessary data fetching.